sdk/network/Azure.ResourceManager.Network/samples/Generated/Samples/Sample_NetworkWatcherResource.cs

```diff
@@ -212,7 +212,7 @@ public async Task GetNextHop_GetNextHop()
         public async Task GetVmSecurityRules_GetSecurityGroupView()
         {
             // Generated from example definition: specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/examples/NetworkWatcherSecurityGroupViewGet.json
-            // this example is just showing the usage of "NetworkWatchers_GetVmSecurityRules" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "NetworkWatchers_GetVMSecurityRules" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `VM`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/networkWatcher.json#L546